src/_posts/compliance/2000-01-01-hds.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Hébergeur de Données de Santé - Health Data Hosting Provider 
+nav: HDS
+modified_at: 2022-09-12 00:00:00
+tags: compliance hds
+---
+
+*(version française [ci-dessous](#fr-has-scalingo-an-hds-certification))*
```

Comment:
I think you want to use `-quest-ce-que-la-certification-iso-27001-` for the first french item. Same for second link

Reply:
You're right ! Fixed